### PR TITLE
[test] use explicit functions

### PR DESCRIPTION
**Pull request by nsbradford**

### CODE REVIEW

**Comment by nsbradford** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #2** <a href="https://app.graphite.dev/github/pr/ellipsis-dev/TalkFormAI/2?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ellipsis-dev/TalkFormAI/2?utm_source=stack-comment).

File: src/utils.ts [hunk 1]
```diff
@@ -131,9 +131,9 @@ export async function submitResponseToSupabase(
   }
 }
 
-export const removeStartAndEndQuotes = (str: string | null) => {
+export function removeStartAndEndQuotes(str: string | null) {
```
Comment (nsbradford) [inline src/utils.ts:134]: @ellipsis-dev add a docstring. Make sure to run the build command to make sure everything works.

Comment (nsbradford) [inline src/utils.ts:134]: @ellipsis-dev add a docstring. Make sure to run the build command to make sure everything works.

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -11,6 +11,9 @@ import { useState } from 'react';
 import Head from 'next/head';
 config.autoAddCss = false;
 
+const isProduction = process.env.NODE_ENV === 'production';
+const isProduction = process.env.NODE_ENV === 'production';
```
Comment (hbrooks) [inline src/pages/_app.tsx:15]: @ellipsis-dev remove the dup line

**Pull request closed without merging**